ts/features/messages/components/Home/TabNavigationContainer.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useCallback, useEffect } from "react";
+import { StyleSheet, View } from "react-native";
+import PagerView from "react-native-pager-view";
+import { TabItem, TabNavigation } from "@pagopa/io-app-design-system";
+import I18n from "../../../../i18n";
+import {
+  useIODispatch,
+  useIOSelector,
+  useIOStore
+} from "../../../../store/hooks";
+import { shownMessageCategorySelector } from "../../store/reducers/allPaginated";
+import { setShownMessageCategoryAction } from "../../store/actions";
+import {
+  getInitialReloadAllMessagesActionIfNeeded,
+  messageListCategoryToViewPageIndex,
+  messageViewPageIndexToListCategory
+} from "./homeUtils";
+
+const styles = StyleSheet.create({
+  tabContainer: {
+    paddingVertical: 8
+  }
+});
+
+export const TabNavigationContainer = ({
+  pagerViewRef
+}: {
+  pagerViewRef: React.MutableRefObject<PagerView | null>;
+}) => {
+  // console.log(`=== TabNavigationContainer`);
```

Comment:
still a bunch of logging, imho we could also remove it when adding the full new messages_home screens, but I'm not so sure it would be so widely appreciated

Reply:
Same as before